src/init.cpp

```diff
@@ -1241,6 +1236,14 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     GetMainSignals().RegisterBackgroundSignalScheduler(scheduler);
 
+    /* Register RPC commands regardless of -server setting so they will be
+     * available in the GUI RPC console even if external calls are disabled.
```

Comment:
Only relevant in bitcoin-qt? No need to register if bitcoind without server.

Reply:
That's very rarely used. It doesn't hurt to register the commands anyway. If that helps keep the code simpler.

Reply:
Yeah agree.